### DISCUSSION TITLE
Converting old style projects to new Beta Projects

### DISCUSSION THREAD

**Discussion by karina-klinkeviciute** [community/community#9130] [Projects and Issues] [answered]
It would be good to have an easy option to convert/copy an existing project to the new Beta project.

**Comment by dezren39** [comment]
This is exactly what I need.

**Reply by AlexOlza** [reply]
Same here.

**Reply by ghost** [reply]
same :D

**Comment by mirmostafa** [comment]
Any good news? No way yet?

**Comment by johnwdubois** [comment]
Me too! I've invested way too much in my Projects to leave them behind.
It would be ironic if those who have actually used Projects in the past are ignored when creating the "new and improved" Projects beta.

**Comment by omerbensaadon** [comment]
Hey folks, this is actually in the works now! Would anyone be interested in beta testing this? cc: @j0siepy

**Reply by ghost** [reply]
✋🏻

**Reply by 5HT2** [reply]
I would love to be able to beta test this! :tada:

**Comment by j0siepy** [comment]
If you're interested in testing out our migration tool please @ me and I can give you access! :tada:

**Reply by ghost** [reply]
interested by the migration tool @j0siepy 😃

**Reply by johnwdubois** [reply]
I'm also interested by the migration tool. @j0siepy

**Reply by Descent098** [reply]
I would love to as well @j0siepy

**Comment by johnwdubois** [comment]
Me too  @j0siepy <https://github.com/j0siepy>.
Best,
John

=======================================

John W. DuBois
Professor of Linguistics
University of California, Santa Barbara
Santa Barbara, California 93106
USA
Email: ***@***.***
Zoom: Zoom room <https://ucsb.zoom.us/j/9851117049>
Web page: http://www.linguistics.ucsb.edu/faculty/dubois/
Rezonator: https://rezonator.com

**Comment by j0siepy** [comment, marked as answer]
🥳  Everyone can now access the migration tool via feature preview!!


https://user-images.githubusercontent.com/98360703/174348285-c6b7a290-49d4-4bb3-b65d-8893a39f2de0.mov


Here's a link to [documentation](https://docs.github.com/en/issues/trying-out-the-new-projects-experience/migrating-your-project) 
and where you can provide [Feedback](https://github.com/github-community/community/discussions/18752)

Feel free to tag me if you have any questions!

**Comment by jbylund** [comment]
@j0siepy getting a 500 trying to migrate a project, is there a specific place that should be reported? anything else you need to know?
